server/context.go

```diff
@@ -165,12 +166,12 @@ func (ctx *Context) initEngine(attrsStr, path string) (engine.Engine, error) {
 	return engine.NewRocksDB(attrs, path, ctx.CacheSize), nil
 }
 
-// parseAttributes parses a colon-separated list of strings,
+// parseAttributes parses a list of strings separated by sep,
 // filtering empty strings (i.e. ",," will yield no attributes.
 // Returns the list of strings as Attributes.
-func parseAttributes(attrsStr string) proto.Attributes {
+func parseAttributes(attrsStr string, sep string) proto.Attributes {
```

Comment:
Do we need to be able to parse attributes in two different ways? Can we just use the same separator everywhere or is there some reason why we want `:` in some places and `,` in others?